src/renderers/controls/enum.control.tsx

```diff
@@ -42,27 +52,26 @@ export class EnumControl extends Control<ControlProps, ControlState> {
           className={classNames.input}
           hidden={!visible}
           disabled={!enabled}
-          value={this.state.value}
+          value={this.state.value || ''}
```

Comment:
doesn't the value of a `select` control correspond to the key? so shouldn't the alternative be `'empty'` rather than an empty string?